web/src/compose_closed_ui.js

```diff
@@ -67,20 +67,20 @@ function update_reply_button_state(disable = false) {
     $(".compose_reply_button").attr("disabled", disable);
     if (disable) {
         $("#compose_buttons .compose-reply-button-wrapper").attr(
-            "data-tooltip-template-id",
-            "compose_reply_button_disabled_tooltip_template",
+            "data-reply-button-type",
+            "disabled",
         );
         return;
     }
     if (narrow_state.is_message_feed_visible()) {
         $("#compose_buttons .compose-reply-button-wrapper").attr(
-            "data-tooltip-template-id",
-            "compose_reply_message_button_tooltip_template",
+            "data-reply-button-type",
+            "enabled",
```

Comment:
Naming the options of `data-reply-button-type` as `enabled` and `selected-topic` makes it a bit confusing. Makes me wonder if `selected-topic` is enabled? Or some other intermediate state between `enabled` and `disabled`.

We could use something like `direct-disabled`, `selected-message` and `selected-conversation` instead.